lib/cocina_checker.rb

```diff
@@ -7,9 +7,12 @@ def self.check
   end
 
   def check_cocina
-    puts "repos to Cocina check: #{repos.map(&:name).join(', ')}"
+    repo_names = repos.map(&:name)
+    puts "repos to Cocina check: #{repo_names.join(', ')}"
+    version_map = Dir["#{base_directory}**/#{Settings.gem_filename}"].filter_map do |lockfile_path|
+      project_name_for = lockfile_path.gsub(base_directory, '').gsub(Settings.gem_filename, '').chomp('/')
```

Comment:
isn't this the same as the `project_name_for` method?  Please don't choose a variable that shadows a method name.

Reply:
@jcoyne I think @ndushay and I were leaning towards the impl in #97 so this may or may not be worth looking into

Reply:
Whoops, yes, didn't realize that method name already existed...fixed here, but indeed #97 seems to be winning now.